server/utils/enums.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     PRETTY_DATE: 'EEEE d MMMM yyyy',
     PRETTY_DAY: 'EEEE',
     PRETTY_DAY_AND_MONTH: 'd MMMM',
-    PRETTY_TIME: 'h:mmaaa',
+    PRETTY_TIME: 'h:mmaa',
```

Comment:
this was fixed to be respected to actually produce lower case AM and PM. 
https://github.com/date-fns/date-fns/pull/2016/files

That might actually be the desired behaviour but the tests specifically stated upper case values so have maintained existing behaviour